static/js/components/ExpandedPostDisplay.js

```diff
@@ -88,60 +98,91 @@ export default class ExpandedPostDisplay extends React.Component<*, void> {
       beginEditing,
       isModerator,
       showPostDeleteDialog,
-      showPostReportDialog
+      showPostReportDialog,
+      postDropdownMenuOpen,
+      showPostMenu,
+      hidePostMenu,
+      showPostShareMenu,
+      hidePostShareMenu,
+      postShareMenuOpen
     } = this.props
 
     return (
       <div className="post-actions">
-        <PostVotingButtons
-          post={post}
-          className="expanded"
-          toggleUpvote={toggleUpvote}
-        />
-        {SETTINGS.username === post.author_id && post.text ? (
-          <div
-            className="comment-action-button edit-post"
-            onClick={beginEditing(editPostKey(post), post)}
-          >
-            <a href="#">edit</a>
-          </div>
-        ) : null}
-        {post.num_reports ? (
-          <div className="report-count">Reports: {post.num_reports}</div>
-        ) : null}
-        {SETTINGS.username === post.author_id ? (
-          <div
-            className="comment-action-button delete-post"
-            onClick={showPostDeleteDialog}
-          >
-            <a href="#">delete</a>
-          </div>
-        ) : null}
-        {userIsAnonymous() ? null : this.postSubscriptionButton()}
-        {isModerator && !post.removed ? (
-          <div
-            className="comment-action-button remove-post"
-            onClick={this.removePost.bind(this)}
-          >
-            <a href="#">remove</a>
-          </div>
-        ) : null}
-        {isModerator && post.removed ? (
-          <div
-            className="comment-action-button approve-post"
-            onClick={this.approvePost.bind(this)}
-          >
-            <a href="#">approve</a>
-          </div>
-        ) : null}
-        {!userIsAnonymous() ? (
-          <div
-            className="comment-action-button report-post"
-            onClick={showPostReportDialog}
-          >
-            <a href="#">report</a>
+        <div className="left">
+          <PostVotingButtons
```

Comment:
Are you addressing the icon change for the upvote button in a separate PR?

Reply:
The last I heard the design wasn't yet finalized, so I have held off on it so far